build.gradle

```diff
@@ -155,4 +157,17 @@ configure(coreProjects) {
         }
     }
     check.dependsOn jacocoTestCoverageVerification
+}
+
+licenseReport {
+    configurations = ALL
+    excludeOwnGroup = true
+    excludeBoms = false
+    renderers = new TextReportRenderer()
+}
+
+task copyThirdParyReport(type: Copy) {
```

Comment:
I suggest a different name like `generateThirdPartyReport`. This task also needs to depend on the task from the `dependency-license-report` plugin. Otherwise, a developer will need to run two tasks.